plugins/modules/grafana_dashboard.py

```diff
@@ -67,10 +67,11 @@
       - Revision of the public grafana dashboard to import
     default: 1
     version_added: '2.10'
-  message:
+  msg:
```

Comment:
This change should be in an other PR I think..

Reply:
yeah probably, I faced this issue while pushing the tests.
In fact we have to port the changes made in this patch https://github.com/ansible/ansible/pull/60051/files